tpgtools/overrides/firebaserules/samples/release/firestore_secondary_release.tf.tmpl

```diff
@@ -0,0 +1,36 @@
+resource "google_firebaserules_release" "primary" {
+  provider     = google-beta
+  name         = "cloud.firestore/${google_firestore_database.secondary.name}"
+  ruleset_name = "projects/{{project}}/rulesets/${google_firebaserules_ruleset.firestore.name}"
+  project      = "{{project}}"
+
+  lifecycle {
+    replace_triggered_by = [
+      google_firebaserules_ruleset.firestore
+    ]
+  }
+}
+
+# Provision a non-default Firestore database.
+resource "google_firestore_database" "secondary" {
+  project     = "{{project}}"
+  name        = "{{database}"
+  location_id = "{{region}}"
+  type        = "FIRESTORE_NATIVE"
+}
+
+# Create a ruleset of Firebase Security Rules from a local file.
+resource "google_firebaserules_ruleset" "firestore" {
+  provider = google-beta
+  project  = "{{project}}"
+  source {
+    files {
+      name    = "firestore.rules"
+      content = "service cloud.firestore {match /databases/{database}/documents { match /{document=**} { allow read, write: if false; } } }""
```

Comment:
```suggestion
      content = "service cloud.firestore {match /databases/{database}/documents { match /{document=**} { allow read, write: if false; } } }"
```

Reply:
Multiple string formatting issues in the test execution- not sure if it was all caused by this one